tests/resources/importedModelWithCustomUnits1.cellml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<model xmlns="http://www.cellml.org/cellml/2.0#" name="import_me">
+  <units name="myUnits">
+    <unit units="second" prefix="milli" />
+  </units>
+  <component name="importMe">
+    <variable name="x" units="myUnits" interface="public_and_private"/>
+  </component>
+</model>
```

Comment:
This CellML file is not being used anymore. Delete it.

Reply:
47e287cd